.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] [OPTIONAL] please add `.vscode` to your .gitignore since it's a file you don't need in your repo